src/ocamlorg_frontend/pages/home.eml

```diff
@@ -263,22 +263,62 @@ Layout.render
     </div>
   </div>
 </div>
-<div class="bg-legacy-default py-12 overflow-hidden">
-  <div class="container-fluid flex flex-col items-center">
-    <h3 class="font-bold">Releases</h3>
-    <div class="bg-pattern p-6 mt-8 rounded-xl w-full lg:w-2/3 text-white">
-      <a href="<%s Url.release latest_release.version %>">
-        <div class="text-primary text-sm mb-4 uppercase">Recent Release</div>
-        <h3 class="font-bold mb-6">
-          <%s latest_release.version %> <span class="text-sm font-medium">(<%s latest_release.date %>)</span>
-        </h3>
-      </a>
-      <div class="prose prose-sm text-white"><%s! latest_release.highlights_html %></div>
-    </div>
-    <a href="<%s Url.releases %>" class="btn btn-lg btn-ghost mt-8">
-      <span>See all Releases</span>
-      <%s! Icons.chevron_right "h-5 w-5" %>
-    </a>
+<div class="bg-legacy-default pt-24 pb-14 overflow-hidden">
+  <div class="container-fluid flex flex-col md:flex-row gap-10 items-stretch mb-14"> 
+      <div class="basis-1/2">
+        <h3 class="font-bold">Releases</h3>
+        <div class="p-10 mt-7 w-full border-2 shadow-md rounded-2xl h-full">
+          <a href="<%s Url.release latest_release.version %>">
```

Comment:
Another thing I see is that we only have the latest version here, so we'd need to bring in and use a list of recent releases here, or else the box will be very empty.